module/modmanager/manager.go

```diff
@@ -822,6 +848,32 @@ func (mgr *Manager) attemptRestart(ctx context.Context, mod *module) []resource.
 	// No need to check mgr.untrustedEnv, as we're restarting the same
 	// executable we were given for initial module addition.
 
+	slowTicker := time.NewTicker(2 * time.Second)
```

Comment:
[q] is the point of this ticker just to report that the module is still being added?

[minor] let's convert this block to a function since it's used twice. e.g. something like:

```go
func startSlowTicker(...) (cleanup func()) 
```

Reply:
yep, and done